docs/NooBaaNonContainerized/NooBaaCLI.md

```diff
@@ -0,0 +1,603 @@
+- **[List Buckets](#list-buckets)**: Display a list of all existing buckets, including relevant details.
+- **[Delete Bucket](#delete-bucket)**: Remove unwanted or obsolete buckets from the system.
+
+### Add Bucket
+
+The `bucket add` command is used to create a new bucket with customizable options.
+
+#### Usage
+```sh
+noobaa-cli bucket add --name <bucket_name> --owner <owner_name> --path <path> 
+[--bucket_policy][--fs_backend][--force_md5_etag][--from_file]
+```
+#### Flags -
+- `name` (Required)
+    - Type: String
+    - Description: Specifies the name of the new bucket.
+
+- `owner` (Required)
+    - Type: String
+    - Description: Specifies the name of the owner of the bucket.
+
+- `path` (Required)
+    - Type: String
+    - Description: Specifies the underlying path of the bucket. Path must be an absolute path.
+
+- `bucket_policy`
+    - Type: String
+    - Description: Set the bucket policy, type is a string of valid JSON policy
+
+- `fs_backend`
+    - Type: String
+    - Enum: none | GPFS | CEPH_FS | NFSv4
+    - Description: Specifies the file system of the bucket (default config.NSFS_NC_STORAGE_BACKEND)
+
+- `force_md5_etag`
+    - Type: Boolean
+    - Description: Set the bucket to force md5 ETag calculation
+
+- `from_file`
+    - Type: String
+    - Description: Path to a JSON file which includes bucket properties. When using `from_file` flag the bucket details must only appear inside the options JSON file.
+
+
+### Update Bucket
+
+The `bucket update` command is used to update an existing bucket with customizable options.
+
+#### Usage
+```sh
+noobaa-cli bucket update --name <bucket_name> [--new_name] [--owner]
+[--path][--bucket_policy][--fs_backend][--force_md5_etag][--from_file]
+```
+#### Flags -
+- `name` (Required)
+    - Type: String
+    - Description: Specifies the name of the updated bucket.
+
+- `new_name`
+    - Type: String
+    - Description: Specifies the new name of the bucket.
+
+- `owner`
+    - Type: String
+    - Description: Specifies the name of the owner of the bucket.
+
+- `path`
+    - Type: String
+    - Description: Specifies the underlying path of the bucket. Path must be an absolute path.
+
+- `bucket_policy`
+    - Type: String
+    - Description: Set the bucket policy, type is a string of valid JSON policy.
+
+- `fs_backend`
+    - Type: String
+    - Enum: none | GPFS | CEPH_FS | NFSv4
+    - Description: Specifies the file system of the bucket (default config.NSFS_NC_STORAGE_BACKEND), unset with ''.
+
+- `force_md5_etag`
+    - Type: Boolean
+    - Description: Set the bucket to force md5 ETag calculation.
+
+
+### Bucket Status
+
+The `bucket status` command is used to print the status of the bucket.
+
+#### Usage
+```sh
+noobaa-cli bucket status --name <account_name>
+```
+#### Flags -
+- `name` (Required)
+    - Type: String
+    - Description: Specifies the name of the bucket.
+
+### List buckets
+
+The `bucket list` command is used to display a list of all existing buckets.
```

Comment:
Do we also want to filter it by owner? I think it can be useful.

Reply:
currently no